setup.cfg

```
@@ -0,0 +1,32 @@
+[metadata]
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3 :: Only
+license = Apache 2.0
+long_description = file: README.md
+name = dwave-sklearn-plugin
+project_urls =
+    Changes = https://github.com/dwavesystems/dwave-sklearn-plugin/releases
+    Documentation = https://docs.ocean.dwavesys.com
+    Souce Code = https://github.com/dwavesystems/dwave-sklearn-plugin
```

typo

---

Probably should be but it actually seems to work fine. I copied the format from [dwave-samplers](https://github.com/dwavesystems/dwave-samplers/blob/8a3122e1bde8eeb3505b1e4b8a337d66f9173b55/setup.cfg#L15) and it renders correctly on [PyPI](https://pypi.org/project/dwave-samplers/).

---

`Souce Code` renders correctly?

---

It does indeed

---

Ok, maybe diff will clarify it :smile: 
```suggestion
    Source Code = https://github.com/dwavesystems/dwave-sklearn-plugin
```

---

Ha! I am clearly confused. Will fix. Thanks